components/raftstore/src/store/fsm/store.rs

```diff
@@ -24,7 +24,7 @@ use kvproto::replication_modepb::{ReplicationMode, ReplicationStatus};
 use protobuf::Message;
 use raft::{Ready, StateRole};
 use time::{self, Timespec};
-use tokio_threadpool::{Sender as ThreadPoolSender, ThreadPool};
+use tokio::runtime::{Builder as RuntimeBuilder, Handle, Runtime};
```

Comment:
I think it is better to import `self` and use `runtime::Builder` rather than alias to `RuntimeBuilder` (maybe you can just use `Builder` too).